src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -1,49 +1,99 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
+import intersection from 'lodash/intersection';
 
 class Hovercard extends React.Component {
   static propTypes = {
+    displayTimeout: PropTypes.number,
     className: PropTypes.string,
     containerClassName: PropTypes.string,
     header: PropTypes.node,
     body: PropTypes.node,
   };
 
+  static defaultProps = {
+    displayTimeout: 100,
+  };
+
   constructor(...args) {
     super(...args);
     this.state = {
       visible: false,
+      rendered: false,
     };
+    this.hoverWait = null;
+    this.cardElement = null;
   }
 
-  handleToggleHovercard = () => {
-    let {header, body} = this.props;
+  handleToggleOn = () => this.toggleHovercard(true);
+  handleToggleOff = () => this.toggleHovercard(false);
+
+  toggleHovercard = visible => {
```

Comment:
I think it's worth trying to test this function

Reply:
Good call, I'm definitely down to add some tests.

I'll also try and improve the story, since right now it doesn't actually render anything in the acceptance tests.